src/main/resources/static/app/request/edit-request.html

```diff
@@ -603,6 +603,88 @@ <h4 class="h4">{{ 'Data linkage' | translate }}</h4>
                         </div>
                     </div>
 
+                    <!-- Informed consent form  -->
+
+                    <div class="form-group" ng-show="(!request.statisticsRequest && request.linkageWithPersonalData && request.informedConsent)"
+                         ng-class="{ 'has-error' : request.informedConsentFormAttachments.length < 1 }">
+                        <label class="control-label col-sm-3" for="requestTitle">{{ 'Upload informed consent form' | translate }}</label>
+                        <div class="col-sm-8">
+                            <p ng-show="request.informedConsentFormAttachments.length < 1"><em>{{'No informed consent form uploaded.' | translate}}</em></p>
+
+                            <div class="list-group">
+                                <a ng-repeat="file in request.informedConsentFormAttachments"
```

Comment:
I guess that in the `ng-disabled` values of the submit button, there should be a check that `request.informedConsentFormAttachments` is not empty.